elements/lisk-dpos/src/delegates_info.ts

```diff
@@ -158,11 +158,14 @@ export class DelegatesInfo {
 				block.height + 1,
 				stateStore,
 			);
+			const lastBlockHeaders = [
+				...stateStore.consensus.lastBlockHeaders,
+			].reverse();
 
 			const [randomSeed1, randomSeed2] = generateRandomSeeds(
 				round,
 				this.rounds,
-				stateStore.consensus.lastBlockHeaders,
+				lastBlockHeaders,
```

Comment:
this doesn't need to be reversed. `generateRandomSeeds` should just use `lastBlockHeaders[0]` for the last block